vizro-ai/docs/pages/user-guides/customize-vizro-ai.md

```diff
@@ -62,6 +64,7 @@ vizro_ai = VizroAI(model="<chosen model>")
     | Base API URL      | `MISTRAL_BASE_URL`                        |
 
     To use Mistral with Vizro-AI, you can either use their API, which comes with [an associated cost](https://mistral.ai/technology/#pricing), or you could use their models for free under the Apache 2.0 license. In that case you need to setup the model API yourself. You can check [all available Mistral models including pricing on their website](https://docs.mistral.ai/getting-started/models/models_overview). This will also explain which version the below string acronyms currently point to.
+    Before usage, please review Mistral guidelines on risk mitigation to understand potential model limitations and best practices. More details on responsible usage can be found on this [link](https://docs.mistral.ai/capabilities/guardrailing/#guardrailing-results-with-mistral-safety-prompt).
```

Comment:
```suggestion
    Before using a model, please review Mistral guidelines on risk mitigation to understand potential model limitations and best practices. [See the Mistral site for more details on responsible usage](https://docs.mistral.ai/capabilities/guardrailing/#guardrailing-results-with-mistral-safety-prompt).
```

Reply:
The problem with this link is that I am not sure we have that enabled atm? The other two vendors have more general advice, but here it is really about the API call. Should we enable this? And is there potentially more general info?